client/packages/core/src/Reactor.js

```diff
@@ -348,7 +348,17 @@ export default class Reactor {
     this._oauthCallbackResponse = this._oauthLoginInit();
 
     // kick off a request to cache it
-    this.getCurrentUser();
+    this.getCurrentUser().then((userInfo) => {
+      this.syncUserToEndpoint(userInfo.user);
+    });
+
+    setInterval(
+      async () => {
+        const currentUser = await this.getCurrentUser();
+        this.syncUserToEndpoint(currentUser.user);
+      },
+      1000 * 60 * 20,
```

Comment:
- I realize we may want to cancel this interval, in the case that someone calls `destroy` on this Reactor
- imo we should be running this interval like every minute. Running once every 20 minutes imo _could_ work, but imo it will open us up for hard-to-discover bugs. If we instead push every 1 or 2 minutes, whatever bug we have will be easier to discover. 

cc @dwwoelfel if you have differing thoughts